.github/workflows/build.yml

```diff
@@ -70,6 +70,27 @@ jobs:
         run: |
           cd ..
           rm -rf build
+  docker_test:
+    name: "Build Docker environment container"
```

Comment:
This is the same comment as for `docker_env`. What's the difference? (and maybe make it clear in a comment or something)

Reply:
This is temporary, I just need a way of generating a docker image that isn't one that CI or users use

Reply:
Gone